runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -391,7 +391,7 @@ public String execModule(String fileName) {
 			ProcessBuilder builder = new ProcessBuilder(goPath, "run", modulePath,
 					inputPath);
 			builder.environment().put("GOPATH",
-					runtimePath + File.pathSeparator + tmpdir);
+					builder.environment().get("GOPATH") + ":" + runtimePath + File.pathSeparator + tmpdir);
```

Comment:
`$GOPATH` ends up as `:<runtimePath>:<tmpdir>` (I guess), i.e. the first element is the empty string. Go freaks out with an error message along the lines of "GOPATH needs to be absolute, but this is relative" (or something).